theme/templates/collections/collection.html

```diff
@@ -57,7 +57,16 @@ <h3>Tiles</h3>
       </ul>
         {% endif %}
       {% endfor %}
-      {% endif  %}
+      {% endif %}
+      <h3>Output Formats</h3>
+      <ul>
+      {% for link in data['links'] %}
+        {% if link['rel'] == 'items' and link['type'] == 'application/geo+json' %}
+          <li>GeoJSON</li>
+          <li>CSV</li>{# generally supported with GeoJSON #}
```

Comment:
generally supported with pygeoapi feature collections

Reply:
Updated the comment line accordingly.